assets/dev/scss/editor/panel/panel.scss

```diff
@@ -114,6 +119,29 @@
 	}
 }
 
+#elementor-panel-get-pro-elements-sticky {
+	position: sticky;
+	bottom:0;
+	font-size: 13px;
+	font-weight: 500;
+
+	.elementor-get-pro-sticky-message {
+		height: 40px;
+		padding: 9px 35px;
+		gap: 10px;
+		background-color: #FFFFFF;
```

Comment:
You will need to use css variable for the colors, to accommodate the light and dark modes.

Reply:
fixed